src/main/java/com/minecolonies/structures/lib/RenderUtil.java

```diff
@@ -0,0 +1,56 @@
+package com.minecolonies.structures.lib;
+
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.util.Mirror;
+import net.minecraft.util.Rotation;
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+public final class RenderUtil
+{
+
+    private RenderUtil()
+    {
+        throw new IllegalArgumentException("Utility Class");
+    }
+
+    public static void applyRotationToYAxis(@NotNull final Rotation rotation)
+    {
+        float angle = 0f;
+        switch (rotation)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)